### PR TITLE
tune learning rate bahdanau-attention

### LINKED ISSUE #2099: Need to tune performance for MXNet & TensorFlow for seq2seq
http://preview.d2l.ai.s3-website-us-west-2.amazonaws.com/d2l-en/master/chapter_recurrent-modern/seq2seq.html
http://preview.d2l.ai.s3-website-us-west-2.amazonaws.com/d2l-en/master/chapter_attention-mechanisms/bahdanau-attention.html
http://preview.d2l.ai.s3-website-us-west-2.amazonaws.com/d2l-en/master/chapter_attention-mechanisms/transformer.html

We need to tune performance for MXNet & TensorFlow to obtain similar performance of PyTorch for each section, such as learning rate & max_epochs.

### PR DESCRIPTION
**Pull request by AnirudhDagar**
*Description of changes:*
This PR should partially fix #2099 but the TF hyperparams/implementation of the section on seq2seq/transformers needs a follow up to fix. I tried different sets of hyperparams for TF but those didn't yield the expected results.

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by AnirudhDagar** [comment]
@d2l-bot please rebuild. It was failing earlier due to a broken notebook in master.

**Comment by astonzhang** [comment]
Is this still WIP while transformer TF needs fix?

**Comment by astonzhang** [comment]
Is this still WIP while transformer TF needs fix?
